src/i18n/index.tsx

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
+import Backend from 'i18next-xhr-backend';
+import { initReactI18next } from 'react-i18next';
+import en from './locales/en.json';
+import vn from './locales/vn.json';
+
+i18n
+  .use(Backend)
+  .use(initReactI18next)
+  .init({
+    lng: 'en',
```

Comment:
The initial language should be read from `sessionStorage`, as any change of the language is stored in `sessionStorage`, right?. Setting it as `en` like this probably causes the following issue, when the flag is Vietnam but the language is still in English (This occurs when I change the language to VN, then close and restart the app using `yarn start` in a terminal).

![image](https://user-images.githubusercontent.com/67769750/106102271-b8873700-6171-11eb-8854-a23588f89756.png)